src/utils.js

```diff
@@ -268,7 +269,13 @@ const utils = {
     let retValue = false;
 
     for (let i = 0; i < alerts.length; i++) {
-      retValue =  alerts[i].categories.some(a => category.indexOf(a) >= 0);
+      let alertCategories = alerts[i].categories;
+      retValue =  alertCategories.every(
+        a => categories.every(
```

Comment:
@rhenshaw56 I haven't worked on the alert code before, so I don't fully understand this PR, but please the "a" and "b" variable names to something more descriptive.